CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ the detailed section referring to by linking pull requests or issues.
 
 #### Minor
```

Comment:
![image](https://github.com/sovity/edc-ui/assets/30114507/48c9051d-bc0f-48fc-a9c2-031346bf05c9)

Description: from date should be optional, too